src/app/games/sahara/SaharaFailure.ts

```diff
@@ -1,10 +1,10 @@
 export class SaharaFailure {
 
-    public static readonly CAN_ONLY_REBOUNCE_ON_BLACK: string = $localize`On ne peux rebondir que sur les cases foncées.`;
+    public static readonly CAN_ONLY_REBOUNCE_ON_BLACK: string = $localize`You can only rebounce on dark cases.`;
 
-    public static readonly CAN_ONLY_REBOUNCE_ON_EMPTY_CASE: string = $localize`Vous ne pouvez rebondir que sur les cases vides.`;
+    public static readonly CAN_ONLY_REBOUNCE_ON_EMPTY_CASE: string = $localize`You can only rebound on empty cases.`;
 
```

Comment:
bounce ou rebound selon joojle rotation, d'ailleurs drôlerie, il traduit cases "square" et c'est aussi le mot utilisé sur la page du fameuuux jeu d'échec... Ferions nous un francicisme faux-amique ?

Reply:
bien vu ! Sur "draughts" c'est aussi "square", mais sur Dvonn c'est "space". Peut-être tfaut-il utiliser le terme adéquat pour chaque jeu ?

Reply:
Je suis toujours favorable à l'uniformité quand ça fait plus sens que l'inverse.
Du coup faudra choisir aussi entre bounce and rebound, Je vois toujours du rebounce :p